RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py

```diff
@@ -44,5 +44,13 @@
 
 from Configuration.Eras.Modifier_peripheralPbPb_cff import peripheralPbPb
 peripheralPbPb.toModify(seedGeneratorFromRegionHitsEDProducer,
+                        doClusterCheck = True,  # FIXMETOO
```

Comment:
Shouldn't this doClusterCheck parameter remain inside the ClusterCheckPSet?

Reply:
Indeed.  Didn't see it as that era isn't tested in relval.  As discussed with Slava we may merge or remove it soon.  Should I push a fix now, or are you still reviewing?

Reply:
Please do it now,
I was comparing the 92X and 94X versions of your PR, and this is the only significant difference I spotted.